enhancements/network/networking-console-plugin.md

```diff
@@ -0,0 +1,154 @@
+---
+title: networking-console-plugin
```

Comment:
```suggestion
title:console-networking-plugin
```

Reply:
I'm afraid to start a new naming convention here.  Multiple plugins end with `console-plugin`